### ISSUE TITLE
Changing layer_size cannot predict sentences

### ISSUE THREAD

**Issue by Fancy7777** [neubig/lamtram#31]
when I change the layer_size to another value, it gives this error for generating sentences. Any idea? I checked the source code. It seems this is not implemented.

what():  Dimensions of parameter /_3 looked up from file ({128,128}) do not match parameters to be populated ({512,128})

**Comment by liesun1994** [comment]
That's , in my opinion , a minor bug . 
Here you can not change layer_size into other value . 
I am puzzled too . May something wrong with populate .

**Comment by Fancy7777** [comment]
Yes. I do check the code. There are no options listed to change the
layer_siza. However, to balance the layer size and hidden unit size and etc
is kinda essential to my model.

Hmmm are u thinking to update maybe?

kun wang <notifications@github.com>于2017年10月31日 周二19:22写道：

**Comment by liesun1994** [comment]
@Fancy7777 I think the code will not be updated in the soon later . The author is foucusing on dynet . Issues listed in lamtram may not be solved in time . But you know, several months passed after last commit ......

**Comment by Fancy7777** [comment]
Lol. True. Thanks for the response anyway

kun wang <notifications@github.com>于2017年10月31日 周二19:33写道：

**Comment by liesun1994** [comment]
Hello, I solved the problem by changing macros.cc  layer_size == $value,  where the $value is what you want to changed.
